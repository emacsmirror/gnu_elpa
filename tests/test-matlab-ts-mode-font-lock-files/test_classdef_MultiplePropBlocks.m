% -*- matlab-ts -*-
classdef MultiplePropBlocks
    properties (SetAccess = private, GetAccess = {?OtherClass})
        Property1
        Property2
    end
    properties (Abstract)
        Property3
    end
end
