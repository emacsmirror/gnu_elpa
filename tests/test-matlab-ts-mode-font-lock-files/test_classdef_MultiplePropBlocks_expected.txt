c ccc ccccccccc ccc
kkkkkkkk ffffffffffffffffff
    kkkkkkkkkk bttttttttt o BBBBBBBD ttttttttt o boddddddddddbb
        PPPPPPPPP
        PPPPPPPPP
    kkk
    kkkkkkkkkk bttttttttb
        PPPPPPPPP
    kkk
kkk
